samples/python/multiview_calibration.py

```diff
@@ -0,0 +1,81 @@
+import pdb, sys, traceback, cv2 as cv, numpy as np, os, json, argparse
+
+def calibrateFromPoints(pattern_points, image_points, image_sizes, is_fisheye):
+    """
+    pattern_points: NUM_POINTS x 3
+    image_points: NUM_CAMERAS x NUM_FRAMES x NUM_POINTS x 2
+    is_fisheye: NUM_CAMERAS (bool)
+    image_sizes: NUMCAMERAS x [width, height]
+    """
+
+    num_cameras = len(image_points)
+    num_frames = len(image_points[0])
+    visibility = np.zeros((num_cameras, num_frames), dtype=bool)
+    pattern_points_all = [pts for pts in pattern_points]
+    for i in range(num_cameras):
+        for j in range(num_frames):
+            visibility[i,j] = len(image_points[i][j]) != 0
+
+    success, Rs, Ts, Ks, distortions, rvecs0, tvecs0, errors_per_frame, output_pairs = cv.calibrateMultiview(pattern_points_all,
+                image_points, image_sizes, visibility, is_fisheye, cv.USE_INTRINSICS_GUESS=False)
+    print(Rs)
+    print(Ts)
+    print(Ks)
+    print(distortions)
+
+def calibrateFromImages(files_with_images, grid_size, is_fisheye, dist_m):
```

Comment:
the function expects that the board is visible on all images or at least we have images from all cameras. I propose to generate visibility matrix here and relay on file names. Images with the same file name are captured by different cameras in the same time and observe the same pattern.